tests/test_instance_billing_type_check.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) 2024 SUSE Software Solutions Germany GmbH. All rights reserved.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3.0 of the License, or (at your option) any later version.
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library.
+
+from unittest.mock import patch, Mock
+from requests import exceptions
+from instance_billing_flavor_check import utils
+
+
+@patch('instance_billing_flavor_check.utils.requests.get')
+def test_make_request_ipv4(mock_request_get):
+    """Test make request with IPv4 without issues."""
+    response = Mock()
+    response.status_code = 200
+    response.json.return_value = {'flavor': 'amazing flavor'}
+    mock_request_get.return_value = response
+    assert utils.make_request('1.1.1.1', 'foo', 'bar') == 'amazing flavor'
+    mock_request_get.assert_called_once_with(
+        'https://1.1.1.1/api/instance/check',
```

Comment:
The address `1.1.1.1` belongs to Cloudflare. I'd use documentation address block, for example `203.0.113.1` address. See [RFC5737](https://datatracker.ietf.org/doc/html/rfc5737).